src/api/app/views/webui/requests_listing/_breadcrumb_items.html.haml

```diff
@@ -1,17 +1,10 @@
 - if Flipper.enabled?(:request_index, User.session)
-  - if action_name == 'index'
+  - if @project
+    %li.breadcrumb-item.text-word-break-all
+      %i.fa.fa-cubes
+      = link_to @project, project_show_path(@project)
     %li.breadcrumb-item.active{ 'aria-current' => 'page' }
       Requests
   - else
-    %li.breadcrumb-item
-      = link_to('Requests', requests_path)
-- if @bs_request.present?
-  :ruby
-    current_page = current_page?(request_show_path(@bs_request.number, params[:request_action_id])) ||
-                  current_page?(request_build_results_path(@bs_request.number, params[:request_action_id])) ||
-                  current_page?(request_rpm_lint_path(@bs_request.number, params[:request_action_id])) ||
-                  current_page?(request_changes_path(@bs_request.number, params[:request_action_id])) ||
-                  current_page?(request_mentioned_issues_path(@bs_request.number, params[:request_action_id]))
-  - if current_page
     %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-      Request #{@bs_request.number}
+      Requests
```

Comment:
No need to duplicate this, if it's displayed either way, just drag it out of the if condition